general/community.rst

```diff
@@ -0,0 +1,49 @@
+Community
+=========
+
+Telegram groups
+---------------
+
+    **UBports Supergroup**
+
+        Keep up to date with everything that is happening in the UBports community. Our Telegram Supergroup has nearly 1000 members from around the world and is gowing daily. Ubuntu Touch wouldn't exist without the community and the countless hours our volunteers donate. Thank you to everyone who is using, developing and sharing Ubuntu Touch!
+
+        Join our Telegram group by pressing `here <https://t.me/ubports/>`__!
+
+    **Welcoming Room**
+
+        We invite you to join our "Newcomers Room' where all of the friendly UBports Welcoming Team are ready to help. This is a special area to not only help with installation questions, but also to introduce yourself and get to know some other community members outside our main Supergroup.  We understand that with 1000 members strong, you may not feel comfortable jumping right in, so we created a side area for introductions.  If you would like to join the Ubuntu Touch community this is a great place to start.  
+
+        The UBports Welcoming Team is an excited group of volunteers from all areas of UBports.  Join the group to find out more by pressing `here <https://t.me/joinchat/Baj4lhJ4XRotoeGk1QxuRQ/>`__!
+
+Areas of Interest
+-----------------
+
+    **Latest News**
+        For any kind of news follow our `blog <https://ubports.com/blog/latest-news-1>`__
+        where you can stay up-to-date on everything we do.  
+
+    **TechTalk**
+        For the more technical focused discussions about Ubuntu Touch go over `here <https://ubports.com/blog/techtalk-2>`__.
+
```

Comment:
Links should describe the content that they link to. If it's possible, could you consider making the headings of these definition lists the links, rather than embedding the links in the text? I'm getting 'here' fatigued.

Reply:
Going to change this one soon.

Reply:
There is no way for others to makes edits while in the state? Git noobie here.